client/blocks/importer/wordpress/upgrade-plan/upgrade-plan-details.tsx

```diff
@@ -1,24 +1,220 @@
 import { recordTracksEvent } from '@automattic/calypso-analytics';
+import config from '@automattic/calypso-config';
 import {
+	calculateMonthlyPriceForPlan,
 	getPlan,
+	Plan,
 	PLAN_BUSINESS,
 	PLAN_BUSINESS_MONTHLY,
 	isMonthly,
 } from '@automattic/calypso-products';
 import { CloudLogo, Button, PlanPrice } from '@automattic/components';
+import { SitePlanPricing } from '@automattic/data-stores';
+import { useSitePlans } from '@automattic/data-stores/src/plans';
 import { Title } from '@automattic/onboarding';
 import { Plans2023Tooltip, useManageTooltipToggle } from '@automattic/plans-grid-next';
+import styled from '@emotion/styled';
 import { useI18n } from '@wordpress/react-i18n';
 import clsx from 'clsx';
+import { useTranslate } from 'i18n-calypso';
 import React, { useState, useEffect } from 'react';
 import ButtonGroup from 'calypso/components/button-group';
 import QuerySitePlans from 'calypso/components/data/query-site-plans';
 import { useSelectedPlanUpgradeMutation } from 'calypso/data/import-flow/use-selected-plan-upgrade';
+import { SitePlanData } from 'calypso/my-sites/checkout/src/hooks/product-variants';
```

Comment:
I think we should import this as a type.
```suggestion
import { type SitePlanData } from 'calypso/my-sites/checkout/src/hooks/product-variants';
```